test/tests/ver-1e/tests

```diff
@@ -1,12 +1,48 @@
 [Tests]
   design = 'Diffusion.md TimeDerivative.md DirichletBC.md'
   issues = '#12'
-  [csv]
+  [ver-1e]
     type = Exodiff
     input = ver-1e.i
-    exodiff = ver-1e_out.e
-    requirement = 'The system shall be able to model transient diffusion through a composite slab with a constant concentration boundary condition.'
+    exodiff = ver-1e_test_exodus.e
+    cli_args = 'Executioner/dt=0.2 Executioner/end_time=50 Outputs/exodus/file_base=ver-1e_test_exodus'
```

Comment:
Could you add a statement in the documentation (like I did for ver-1a) to explain that the test is different from the verification run?